### ISSUE TITLE
Buttons 'DWQueue' and 'Restoring' present under the table of 'Migration' tab, is not keyboard accessible while navigating with tab key.

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/azuredatastudio#25150]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;#BM_SQL Azure Data Studio_Mac_Sept2023;#SQL Azure Data Studio;#A11ySev2;#WCAG2.1.1;#Keyboard;#DesktopApp;#Win11;#Mac

### Environment Details:
Azure Data Studio: Version: 1.48.0-insider (Universal)
Operating System: macOS Sonoma 14.1.1

### Steps to Reproduce:

1. Turn on the Voice over.
2. Install Azure data studio and open.
3. Tab till Extension press enter key and install Azure SQL Migration
4. Tab till Respective server and right click.
5. Tab till 'manage' and press enter key.
6. Tab till Azure SQL Migration and press enter key.
7. Tab till 'migrate to azure sq'l and press enter key.
8. Saved session page will open.
9. Select start new session and give next.
10. Databases for assessment page will open.
11. Select the respective databases and give next.
12. Assessment results and SKU recommendations page will open.
13. Tab till next button and press enter.
14. Target platform and assessment results page will open.
15. Tab till 'select target type' dropdown and select Azure SQL Managed Instance.
16. 
Tab till Next button and press enter.
17. Azure SQL target page will open.
18. Filled the required field and tab till next button and press enter.
19. Azure database Migration service page will open.
20. Fill the required field and tab till next button.
21. Data source configuration page will open.
22. Fill the required field and tab till next button and press enter.
23. Summary page will open.
24. Tab till Start migration button and press enter.
25. Tab till Migration tab.
26. Tab till the controls present under the table.

### Actual:
Buttons 'DWQueue' and 'Restoring' present under the table of 'Migration' tab, is not keyboard accessible while navigating with tab key.
Focus after the Ascending checkbox button, it is going over the whole table. After pressing tab again, it is going over the Problems tab.

### Expected:
Buttons 'DWQueue' and 'Restoring' present under the table of 'Migration' tab, should be keyboard accessible while navigating with tab key.

### User Impact:
Keyboard users will not be able to access the button present under the table of 'Migration' tab.

### Attachment: 

[ScreenRecordingForKeyboardFocus.zip](https://github.com/microsoft/azuredatastudio/files/13777556/ScreenRecordingForKeyboardFocus.zip)

**Comment by kisantia** [comment]
@BhanuPrakashGit the attachment is empty. Can you please add screenshots or a new attachment?

**Comment by BhanuPrakashGit** [comment]
GithubTags:#Regressed:01-15-24;#Closed;
Verified the bug in the environment:
Version: 1.48.0-insider (Universal)
Commit: e5e1204d4fb4d4e880cdd893347752ee5736459a
Date: 2023-12-05T06:38:01.191Z
VS Code: 1.82.0
Electron: 25.9.6
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0

Issue is fixed, we are able to navigate over the button with the help of the arrow keys. Hence closing the bug.
PFA for reference

[25150_FocusIssue.zip](https://github.com/microsoft/azuredatastudio/files/13937216/25150_FocusIssue.zip)
